### PR TITLE
[Issue #1957]: Default sortby to posted date descending

### LINKED ISSUE #1957: [Task]:  Sort by Posted Date desc by default
### Summary

More useful to users to sort by Posted Date desc be default

### Acceptance criteria

- Default API call to Posted Date desc
- Posted Date desc comes by default first in the list
- Posted Date asc is 2nd (move them both up)

### PR DESCRIPTION
**Pull request by rylew1**
## Summary
Fixes #1957

## Changes proposed
- default sort by to Posted Date descending

https://github.com/HHS/simpler-grants-gov/assets/93001277/183b5241-aeac-4e62-8d4f-fe371963c91c

### CODE REVIEW

File: frontend/src/app/api/SearchOpportunityAPI.ts [hunk 1]
```diff
@@ -44,6 +44,7 @@ export default class SearchOpportunityAPI extends BaseApi {
     }
 
     const subPath = "search";
+    console.log("requestBody => ", requestBody);
```
Comment (acouch) [inline frontend/src/app/api/SearchOpportunityAPI.ts:47]: This slipped in, should be removed 😉

File: frontend/src/components/search/SearchSortBy.tsx [hunk 2]
```diff
@@ -7,16 +7,16 @@ type SortOption = {
...
-  { label: "Opportunity Title (Ascending)", value: "opportunityTitleAsc" },
+  { label: "Opportunity Number (Ascending)", value: "opportunityNumberAsc" },
   { label: "Opportunity Title (Descending)", value: "opportunityTitleDesc" },
-  { label: "Agency (Ascending)", value: "agencyAsc" },
+  { label: "Opportunity Title (Ascending)", value: "opportunityTitleAsc" },
   { label: "Agency (Descending)", value: "agencyDesc" },
-  { label: "Posted Date (Ascending)", value: "postedDateAsc" },
-  { label: "Posted Date (Descending)", value: "postedDateDesc" },
-  { label: "Close Date (Ascending)", value: "closeDateAsc" },
+  { label: "Agency (Ascending)", value: "agencyAsc" },
```
Comment (rylew1) [inline frontend/src/components/search/SearchSortBy.tsx:17]: @andycochran fyi the new order

Comment (rylew1) [reply frontend/src/components/search/SearchSortBy.tsx:17]: all descending variants come before ascending now

Comment (andycochran) [reply frontend/src/components/search/SearchSortBy.tsx:17]: 🤔 IDK if Z–A should come before A–Z. The date ones should be descending first. But the others makes more sense as ascending first. Can we also change them to: 

"Posted Date (newest)" `// descending`
"Posted Date (oldest)" 
Close Date (upcoming) `// descending`
Close Date (oldest)
Opportunity Title (A to Z) `// ascending`
Opportunity Title (Z to A)
Agency (A to Z)`// ascending`
Agency (Z to A)
Opportunity Number (ascending)
Opportunity Number (descending)

@crystabelrangel, do you have any input on ^these labels?

Comment (rylew1) [reply frontend/src/components/search/SearchSortBy.tsx:17]: What about for posted date something like `Posted Date (newest/descending)`

Comment (andycochran) [reply frontend/src/components/search/SearchSortBy.tsx:17]: However, the results set might not necessarily have any results with close dates in the future. Maybe just newest/oldest is fine, and we ignore the fact that "newest" may be in the future.

Comment (andycochran) [reply frontend/src/components/search/SearchSortBy.tsx:17]: Let's use: 

Posted Date (newest)
Posted Date (oldest)
Close Date (newest)
Close Date (oldest)
Opportunity Title (A to Z)
Opportunity Title (Z to A)
Agency (A to Z)
Agency (Z to A)
Opportunity Number (ascending)
Opportunity Number (descending)

^ in that order

Comment (rylew1) [reply frontend/src/components/search/SearchSortBy.tsx:17]: thanks folks - will try to get to this soon!

Comment (rylew1) [reply frontend/src/components/search/SearchSortBy.tsx]: image

**Pull request closed without merging**